File: C#/백준/Bronze/9635. Balloons Colors/Balloons Colors.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            for (int i=0;i<a;i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int n = aa[0];
                if (bb[0]==aa[1] && bb[n-1]==aa[2]){Console.WriteLine("BOTH");}
                if (bb[0]==aa[1] && bb[n-1]!=aa[2]){Console.WriteLine("EASY");}
                if (bb[0]!=aa[1] && bb[n-1]==aa[2]){Console.WriteLine("HARD");}
                if (bb[0]!=aa[1] && bb[n-1]!=aa[2]){Console.WriteLine("OKAY");}
            }
        }
    }
}

File: Python/백준/Bronze/9635. Balloons Colors/Balloons Colors.py
q = int(input())
for i in range(q):
    n,a,b = map(int,input().split())
    aa = list(map(int,input().split()))
    if aa[0]==a and aa[n-1]==b:
        print('BOTH')
    if aa[0]!=a and aa[n-1]!=b:
        print('OKAY')
    if aa[0]==a and aa[n-1]!=b:
        print('EASY')
    if aa[0]!=a and aa[n-1]==b:
        print('HARD')